packages/i18n/src/lang/whitelist.yml

```diff
@@ -17,3 +17,4 @@
 - z0JksZ # Inuit
 - gTB9r8 # Inuk (Inuit)
 - uNArwM # '+123243234' - Placeholder for form telephone field
+- ybzpGb # temporary output Api message to client
```

Comment:
Appears to have done nothing so I added it to `i18n/src/lang/fr.json`

Reply:
Yeah, I've noticed this, too.  I'm not sure exactly what the expected behaviour of this file is.